dockerfiles/README.md

```diff
@@ -246,6 +247,41 @@ The Dockerfile used in these instructions specifically targets Raspberry Pi 3/3+
     ```
 10. Test installation by following the instructions [here](https://microsoft.github.io/onnxruntime/)
 
+## NVIDIA Jetson TX1/TX2/Nano/N:
+
+These instructions are for [JetPack SDK 4.4](https://developer.nvidia.com/embedded/jetpack).
+The Dockerfile.jetson is using [NVIDIA L4T 32.4.3](https://developer.nvidia.com/embedded/linux-tegra) as base image.
+You can probably used those instructions with earlier/later versions with appropriate modifications.
+Instructions assume you are on Jetson host in the root of onnxruntime git project clone.
+
+Two-step installation is required:
+
+1. Build Python 'wheel' for ONNXruntime on host Jetson system;
+2. Build Docker image using ONNXruntime wheel from step 1. You can also install the wheel on the host directly.
+
+Here are the build commands for each step:
+
+1.1 Install ONNXruntime build dependencies on Jetpack 4.4 host:
+```
+   sudo apt install -y --no-install-recommends \
+    	build-essential software-properties-common cmake libopenblas-dev \
+	libpython3.6-dev python3-pip python3-dev
+```
+1.2 Build ONNXRuntime Python wheel:
+```
+   ./build.sh --update --config Release --build --build_wheel \
+   --use_cuda --cuda_home /usr/local/cuda --cudnn_home /usr/lib/aarch64-linux-gnu
```

Comment:
>--use_cuda [](start = 3, length = 10)

Is this for CUDA EP only? the docker file should be able to support TensorRT also. why not make it another option to build TensorRT also?
Should also mention the TRT + CUDA + cudnn version combinations for Jetpack 4.4.

Reply:
yes it's purposely for CUDA EP only. TRT 7.1 support is not yet officially integrated for ORT.